### PR TITLE
Fixed some typos, changed imports to be the same as other DOCs

### PR DESCRIPTION
**Pull request by mcrowder65**

### CODE REVIEW

File: docs/BottomNavigation.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 ```js
 ...
-import { BottomNavigation } from 'react-native-material-ui';
+import { BottomNavigation } from '../react-native-material-ui';
```
Comment (xotahal) [inline docs/BottomNavigation.md]: It should be just `react-native-material-ui`. Why do you think it should be relative path?

Comment (mcrowder65) [reply docs/BottomNavigation.md]: I think it should just be `react-native-material-ui`. I was just following the other docs

Comment (mcrowder65) [reply docs/BottomNavigation.md]: I will update :)

Comment (mcrowder65) [reply docs/BottomNavigation.md]: Updated. and made all the others the same.

**Pull request merged**